cmd/temporalite/main.go

```diff
@@ -146,6 +149,10 @@ func buildCLI() *cli.App {
 					EnvVars: []string{config.EnvKeyConfigDir},
 					Value:   "",
 				},
+				&cli.StringSliceFlag{
+					Name:  dynamicConfigValueFlag,
+					Usage: `dynamic config value, as KEY=JSON_VALUE (meaning strings need quotes)`,
+				},
```

Comment:
Should we add a `dynamic-config-file` flag as well? That would make it easier to share dynamic config profiles between multiple Temporalite (and regular Temporal server) instances.

I also wouldn't be mad if we added a shorthand `dcv` alias for this.

Reply:
The dynamic config file can be set in the config file itself. We need to make the config file part more robust rather than accepting dynamic config directly (same for all sorts of other config options, of which dynamic config file is just one).